File: src/main/java/com/p2p/lending/service/LoggerService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Log;

import com.github.pagehelper.PageInfo;

public interface LoggerService {

	PageInfo<Log> findList(Log log, Integer pageNo, Integer pageSize);

	public List<Log> findList(Map<String, Object> map);

	void create(Log log);
}

File: src/main/java/com/p2p/lending/service/LoggerService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Borrowmoney
import com.p2p.lending.entity.Log

import com.github.pagehelper.PageInfo

interface LoggerService {

    fun findList(log: Log, pageNo: Int?, pageSize: Int?): PageInfo<Log>

    fun findList(map: Map<String, Any>): List<Log>

    fun create(log: Log)
}
